moxy-compiler/src/main/java/moxy/compiler/viewstate/ViewInterfaceProcessor.kt

```diff
@@ -213,9 +214,9 @@ class ViewInterfaceProcessor(
      * Combines methods, comparing by [ViewMethod.equals], discarding duplicates from superinterface
      */
     private fun combineMethods(
-        methods: Set<ViewMethod>,
-        superInterfaceMethods: Set<ViewMethod>
-    ): Set<ViewMethod> {
+        methods: List<StockViewMethod>,
+        superInterfaceMethods: List<StockViewMethod>
+    ): List<StockViewMethod> {
```

Comment:
I think in the end `combineMethods()` should operate on `Sets`, not on `Lists`, to remove duplicates of `ViewMethods`